Components/Alerts/alerts.css

```diff
@@ -0,0 +1,32 @@
+.alert{
+    min-width: 25rem;
+    display: flex;
+    align-items: center;
+    padding: 0rem 1rem;
+    justify-content: space-between;
+    border-radius: 10px;
+}
+
+.alert > *{
+    color: black;
+}
+
+.alert-info{
+    background-color: var(--info);
+}
+
+.alert-success{
+    background-color: var(--success);
+
+}
+
+.alert-danger{
+    background-color: var(--danger);
+
```

Comment:
Remove extra blank lines

Reply:
I think prettier hasn't done its job. I'll fix it.